ProcessQueue/__init__.py

```diff
@@ -97,7 +106,7 @@ def process_scan_report_sheet_table(sheet):
 
 def main(msg: func.QueueMessage):
     logging.info("Python queue trigger function processed a queue item.")
-
+    print(datetime.utcnow().strftime("%H:%M:%S.%fZ"))

```

Comment:
print check left by mistake

Reply:
I'm leaving these in for now in the (?vain) hope they will give us some useful guidance on timings in production.